scripts/create-secrets/k8sExecution.js

```diff
@@ -170,7 +170,7 @@ function doK8sExecution(config, shouldNotAsk = false) {
         }
 
         createSecret(env, namespace, "auth-secrets", {
-            "jwt-secret": pwgen(),
+            "jwt-secret": pwgen(512),
```

Comment:
This seems produces a 512 chars key (4096 bits) which is not recommended as it's greater than block size.
if we want 512 bits, it should be `64` chars.
https://security.stackexchange.com/questions/95972/what-are-requirements-for-hmac-secret-key

Reply:
Oh yeah :grimacing: